piqe_ocp_lib/api/resources/ocp_projects.py

```diff
@@ -119,6 +119,36 @@ def delete_a_namespace(self, namespace_name):
             logger.error("Exception when calling method delete_a_namespace: %s\n" % e.reason)
         return api_response
 
+    def delete_labelled_projects(self, label_name):
```

Comment:
Can you please add typing, example -  def get_package_manifest(self, package_name: str) -> Optional[ResourceInstance]:

Reply:
I noticed all the other fns as well as other modules do not have the typing. Should this be addressed in this PR?